README.md

```diff
@@ -194,7 +194,30 @@ Explore the [API documentation](docs/api.md) and [examples](https://github.com/G
 
         await page.evaluate(() => console.log(`url is ${location.href}`));
 
-4. Enable verbose logging - All public API calls and internal protocol traffic
+4. Stop test execution and use a debugger in browser
+
+  - Use `{devtools: true}` when launching Puppeteer:
+      ```js
```

Comment:
can you take off the 'js" syntax highlighting. It messes up our import hosting the official docs: https://developers.google.com/web/tools/puppeteer/debugging

Reply:
Sure, is it better now?